content/chainguard/chainguard-images/getting-started/php.md

```diff
@@ -223,6 +223,111 @@ In such cases, the last `FROM` section from the Dockerfile is the one that compo
 
 It's worth highlighting that nothing is carried from one stage to the other unless you copy it. That facilitates creating a slim final image with only what's necessary to execute the application.
 
+## 3. Working with the PHP-FPM Image Variant
+
+The `latest-fpm` image variant is suitable for running PHP applications over FastCGI, to be server via a web server such as Nginx. In this section, we'll run a Docker Compose setup using the `latest-fpm` image variant and the Chainguard Nginx image.
+
+The `namegen-api` demo is a variation of the previous demo, but it serves the random name generation over HTTP. The application responds with a JSON payload containing the animal and adjective combination, and accepts an optional `animal` parameter to specify the animal for the final suggested name.
+
+Start by accessing the `namegen-api` demo folder:
+
+```shell
+cd edu-images-demos/php/namegen-api
```

Comment:
this command doesn't exactly work from the working directory I'm in from the previous step. I think it's fine as is (readers should be able to figure it out) but it's a little funky

Reply:
changed slightly to add clarity! thanks mark!